pkg/addons/data/ClusterRole.rbac.authorization.k8s.io/geneva-logging.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: geneva-logging
```

Comment:
make it less powerfull

Reply:
try adding the SA to the SCC and avoiding the new role and rolebinding and see if that works